robot_smach_states/src/robot_smach_states/rise.py

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env python
+# ROS
+
+import smach
+
+from robot_skills.util.kdl_conversions import VectorStamped
+
+
+class HeroHmiPose(smach.State):
+    """
+    State to ensure that hero's arm is out of the way so that the robot
+    may move his torso up.
+    :param robot: Robot to execute state with
+    """
+    def __init__(self, robot):
+        smach.State.__init__(self, outcomes=['succeeded', 'failed'])
+        self._robot = robot
+
+    def execute(self, _):
+        rotation = 1.57
+        rot_speed = 1
+        arm = self._robot.get_arm()
+
+        #get position to look at
+        goal = VectorStamped(1.0, 0.0, 1.6, frame_id="/" + self._robot.robot_name + "/base_link")
```

Comment:
This is the exact same pose as ```look_at_standing_person```, why not just use ```self._robot.head.look_at_standing_person()```?

Reply:
look_at_standing_person() is defined with respect to the base of the robot. Since we are turning the robot sideways we need to take the point and project it to map frame. 
We only want to move the head after turning is complete, otherwise the robot won't look at the person.

Reply:
Ah, alright! Perhaps add a comment clarifying this :)

Reply:
Ah, alright! Perhaps add a comment clarifying this :)

Reply:
Added a comment for clarification